specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2023-09-01-preview/appplatform.json

```diff
@@ -12311,6 +12311,22 @@
       },
       "x-ms-discriminator-value": "Jar"
     },
+    "WarUploadedUserSourceInfo": {
+      "description": "Uploaded War binary for a deployment",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/JarUploadedUserSourceInfo"
```

Comment:
UploadedUserSourceInfo?

Reply:
war file deployment has all the properties of JarUploadedUserSourceInfo, ex: runtimeVersion & jvmOptions.

Are you proposing to use UploadedUserSourceInfo and add runtimeVersion & jvmOptions as properties in WarUploadedUserSourceInfo?

Reply:
Yeah, please use UploadedUserSourceInfo and add runtimeVersion & jvmOptions as properties in WarUploadedUserSourceInfo